src/Accounts/Accounts/Context/ClearAzureRmContext.cs

```diff
@@ -84,6 +87,15 @@ void ClearContext(AzureRmProfile profile, RMProfileClient client)
                     profile.TrySetDefaultContext(defaultContext);
                     result = true;
                 }
+
```

Comment:
Maintain a list of registered shared components in common.
Clear shared components in one shot.
maybe?